README_PODMAN.md

```diff
@@ -17,25 +17,46 @@ Using your package manager, install the *dnsmasq* package.  For Fedora, this wou
 ## Build and install
 
 1. using git, clone the *github.com/containers/dnsname* repository.
-2. make install PREFIX=/usr -- this will install the dnsname plugin into /usr/libexec/cni where your CNI plugins
-should already exist.
+2. make install PREFIX=/usr -- this will install the dnsname plugin into /usr/libexec/cni where
+your CNI plugins should already exist.
 
 ## Configure a CNI network for Podman
 
 1. Create a new network using `podman network create`.  For example, `podman network create foobar` will suffice.
 
-The following example [configuration file](example/cni-podman1.conflist) shows a usable example for Podman.
+The following example [configuration file](example/foobar.conflist) shows a usable example for Podman.
+
+2. (optional)+The configuration will be automatically enabled for newly created networks via
+`podman network create`. If you want to add this feature to an exisiting network add the needed
+lines to `/etc/cni/net.d/foobar.conflist` using your favorite editor. For example:
+
+ ```
+{
+   "cniVersion": "0.4.0",
+   "name": "foobar",
+   "plugins": [
+      ...
+      {
+	 "type": "dnsname",
+	 "domainName": "dns.podman"
```

Comment:
"capabilities": {
            "aliases": true
         }

add this to the stanza